core/server/src/main/webapp/general.jsp

```diff
@@ -114,6 +114,38 @@
       </div>
     </div>
   </div>
+  <!--  show inconsistent path  -->
+  <% if ((Integer) request.getAttribute("inconsistentPaths") != 0) { %>
+  <div class="row-fluid">
+    <div class="accordion span14" id="accordion4">
+      <div class="accordion-group">
+        <div class="accordion-heading">
+          <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordion4" href="#data4">
+            <h4>Inconsistent Files Details</h4>
+          </a>
+        </div>
+        <div id="data4" class="accordion-body collapse in">
+          <div class="accordion-inner">
+            <table class="table table-hover table-condensed">
+              <thead>
+                <th> <font color="red">On Startup, <%= request.getAttribute("inconsistentPaths") %> inconsistent files were fount. The following files may be corrupted: </font></th>
```

Comment:
@calvinjia  ,  Hi,  I've fixed it.

![image](https://cloud.githubusercontent.com/assets/6999199/21834504/831427ae-d7f0-11e6-982b-b884f1f12001.png)